README.md

```diff
@@ -1,102 +1,97 @@
-
-Serpent Installation:
-```
-git clone https://github.com/ethereum/serpent.git
-cd serpent
-git checkout develop
-sudo pip install pytest
-sudo make && sudo make install
-sudo python setup.py install
-```
-
-Now we can try running some tests to make sure our installation worked.
+Now we can try running some tests to make sure our installation worked:
 
-Run some tests:
 ```
 cd tests
 python runtests.py
 ```
 
-
-# Docker
+## Docker
 
 You may run augur-core using docker as follows:
 
 ### Build:
+
 ```
 docker image build --tag augur-core-tests --file Dockerfile-test .
 ```
 
 ### Run:
+
 ```
-docker container rm -f augur-core-tests; docker container run --rm -it --name augur-core-tests augur-core-tests
+docker container rm -f augur-core-tests
+docker container run --rm -it --name augur-core-tests augur-core-tests
 ```
 
 ### Debug:
+
 ```
-docker container rm -f augur-core-tests; docker container run --rm -it --name augur-core-tests --entrypoint /bin/bash augur-core-tests
+docker container rm -f augur-core-tests
+docker container run --rm -it --name augur-core-tests --entrypoint /bin/bash augur-core-tests
 py.test -s tests/trading_tests.py
 ```
 
+## Additional notes
 
-# Additional Notes:
-
-There are no floats in the serpent language.
-In cases where strings are used, they are actually stored numerically as integers.
-
-All augur-core contracts use fixedpoint (no floats).  So sub-ether values in serpent would be represented as integers whose value
-is in wei (attoEthers or 10**-18 Ethers).
+There are no floats in Serpent.
 
-To give an example, 200\*base / 5 would be 40 in that base.  To multiply two fixed point numbers like 5 times 10 an example in 
-base 10\*\*18 would be 5\*10\*\*18 \* 10\*10\*\*18 / 10\*\*18
-[we divide by the base to keep it in base 10\*\*18].  For a division example, 18/10 would be 18\*10\*\*18 \* 10\*\*18 / (10\*10\*\*18).
+Strings are stored numerically as integers.
 
-Also note, Mist uses a different contract abi style than usual, to convert use the following regex \((.\*?)\) and replace with nothing [except don't do it for events].
+All augur-core contracts use fixedpoint (no floats). So sub-ether values in
+Serpent would be represented as integers whose value is in wei (attoEthers or
+10**-18 Ethers).
 
-# Resources to learn about serpent
+To give an example, 200\*base / 5 would be 40 in that base.
 
-https://github.com/ethereum/wiki/wiki/Serpent
+To multiply two
+fixed point numbers like 5 times 10 an example in base 10\*\*18 would be
+5\*10\*\*18 \* 10\*10\*\*18 / 10\*\*18 (we divide by the base to keep it in
+base 10\*\*18).
 
-https://www.cs.umd.edu/~elaine/smartcontract/guide.pdf
+For a division example, 18/10 would be 18\*10\*\*18 \* 10\*\*18 /
+(10\*10\*\*18).
 
-# More on Augur / high level
+Also note, Mist uses a different contract abi style than usual, to convert use
+the regex `((.*?))` and replace with nothing (except don't do it for
+events).
```

Comment:
It is unclear how this is relevant to augur-core?  I recognize that you just copied this over, but perhaps we should just delete it?  cc @joeykrug